modules/utils/claws/utils/handle_types.hpp

```diff
@@ -1,7 +1,3 @@
-///
-/// *Defined in "HandleTypes.hpp"*
-/// (TODO: rename to handle_types.hpp)*
-///
```

Comment:
Good idea to get rid of the todo. However the part above is a doc comment, it should stay.

Reply:
It just links the source file, which allows people to easily go and check the implementation

Reply:
(Doc comments are preceded by `///` or `/**`)

Reply:
Doc was apparently broken (#37 )